wfe/wfe.go

```diff
@@ -938,6 +938,9 @@ func (wfe *WebFrontEndImpl) Challenge(
 	logEvent *web.RequestEvent,
 	response http.ResponseWriter,
 	request *http.Request) {
+	if features.Enabled(features.RetryAfter) {
+		response.Header().Set("Retry-After", "3")
```

Comment:
Thinking about this PR more, would it be easier to implement this as a configuration item that specifies the retry-after value as "3"? That would sidestep the need for a feature flag and would also reduce the duplication of this value across the WFE/WFE2, making it easier to adjust.